src/test/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderEmploymentRepositoryTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.repository;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.test.context.ActiveProfiles;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace.NONE;
+
+@DataJpaTest
+@ActiveProfiles("test")
+@AutoConfigureTestDatabase(replace = NONE)
+public class OffenderEmploymentRepositoryTest {
+
+    @Autowired
+    private OffenderEmploymentRepository repository;
+
+    @Test
+    public void findAllByNomisId() {
+        var employments1 = repository.findAllByNomisId("G8346GA", Pageable.unpaged());
+        var employments2 = repository.findAllByNomisId("G2823GV", Pageable.unpaged());
+
+        assertThat(employments1).hasSize(3);
```

Comment:
can't see any test for the data being loaded - where do you assert that you've mapped the database columns to the correct fields?